### ISSUE TITLE
Making -f elapsed or timestamp as default when -r is given

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#46]
As commited in b12ffd5, users have to use `-r`/`--time-range` option as follows:
```
  $ uftrace replay -f elapsed -r +8us~
  #  ELAPSED    FUNCTION
     8.174 us |         getpid();
     9.585 us |       } /* c */
     9.845 us |     } /* b */
    10.022 us |   } /* a */
    10.191 us | } /* main */
```

But I think it'd be better to make `-f elapsed` or `-f timestamp` as default depending on `RANGE` stlye. For example, the above example can be simplified as follows:
```
  $ uftrace replay -r +8us~
  #  ELAPSED    FUNCTION
     8.174 us |         getpid();
     9.585 us |       } /* c */
     9.845 us |     } /* b */
    10.022 us |   } /* a */
    10.191 us | } /* main */
```

**Comment by honggyukim** [comment]
As far as I remember, `RANGE` can be given as timestamp range or elapsed time range. But I don't find the clear description in man page. Please have a look at it!

https://github.com/namhyung/uftrace/commit/676e19027867b04866fff31651f20c742b68b3e6#diff-9b4a7c40be3bdadd757ed00bfa1e4281R47

**Comment by honggyukim** [comment]
One more comment is that it's also better to add `-f elapsed` or `-f timestamp` field on top of original default replay fields, not just getting rid of the other fields. @namhyung, please let me know how you think.

**Comment by namhyung** [comment]
Ah, right.  I missed the syntax of elapsed time range in the doc.  Please feel free to send a patch. :)

And I don't agree to make it have same output field as range format.  it might be useful but I think it depends on what user wants.  And I think it makes sense to support a syntax to append or prepend output fields easily.  For example, Linux `perf` tool uses '+' sign to append a sort key.  Maybe something like `-f +elapsed`.

**Comment by honggyukim** [comment]
> Ah, right. I missed the syntax of elapsed time range in the doc. Please feel free to send a patch. :)

Please review https://github.com/namhyung/uftrace/pull/47. :)

> And I don't agree to make it have same output field as range format. it might be useful but I think it depends on what user wants.

Okay. I see.

> And I think it makes sense to support a syntax to append or prepend output fields easily. For example, Linux perf tool uses '+' sign to append a sort key. Maybe something like -f +elapsed.

I agree to add a syntax to append or prepend as you mentioned with `+` notation.

**Comment by namhyung** [comment]
I implemented support for appending fields in d54167f.  I think appending is enough so didn't implement prepending.
